src/NServiceBus.RabbitMQ.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -18,6 +18,7 @@ namespace NServiceBus
         [System.ObsoleteAttribute("Replaced by NServiceBus.Callbacks package. The member currently throws a NotImple" +
             "mentedException. Will be removed in version 5.0.0.", true)]
         public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> DisableCallbackReceiver(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions) { }
+        public static NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> DisableNonNativeTimeouts(this NServiceBus.TransportExtensions<NServiceBus.RabbitMQTransport> transportExtensions, bool allEndpoints) { }
```

Comment:
Yeah, I agree that the naming still needs some work. Your suggestion sounds good.